components/CtaImageContent/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,184 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import "../index.css";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS/Actions/Cta Image Content"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="PageButtons" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <a aria-label="Label" href="https://google.nl/" class="denhaag-cta-image-content">
+      <div class="denhaag-cta-image-content__column denhaag-cta-image-content__image-wrapper">
+        <img
+          class="denhaag-cta-image-content__image"
+          srcset="https://via.placeholder.com/480x270/90D40166 480w,
+                  https://via.placeholder.com/768x432/90D40166 768w,
+                  https://via.placeholder.com/1290x725/90D40166 1290w"
+          src="https://via.placeholder.com/1290x725/90D40166"
+          alt="Label"
+          loading="lazy"
+        />
+      </div>
+      <div class="denhaag-cta-image-content__column">
+        <div class="denhaag-cta-image-content__content">
+          <h3 class="denhaag-cta-image-content__label">Label</h3>
+          <span class="denhaag-cta-image-content__text">Explanational text about the image content link</span>
```

Comment:
For content that is meant to be read out as one long sentence a `span` would make sense, but for these text I think a small pauze between `__text` and `_button` would be appropriate. In cases like this I think you would be better off using a block level element such as `div` instead of the inline `span`.